ts/state/selectors/conversations.ts

```diff
@@ -343,6 +343,22 @@ export const getLeftPaneLists = createSelector(
   _getLeftPaneLists
 );
 
+export const getApprovedConversations = createSelector(
+  getConversationLookup,
+  (lookup: ConversationLookupType): Array<ReduxConversationType> => {
+    return Object.values(lookup).filter(convo => convo.isApproved === true);
+  }
+);
+
+export const getUnapprovedConversations = createSelector(
+  getConversationLookup,
+  (lookup: ConversationLookupType): Array<ReduxConversationType> => {
+    return Object.values(lookup).filter(
+      convo => convo.isApproved === false || convo.isApproved === undefined
+    );
+  }
+);
+
```

Comment:
I think Object.values with filter will break the memoization logic of reselect. Can you have a look is there is another way?